testnets/_IBC/kujiratestnet-terra2testnet.json

```diff
@@ -0,0 +1,31 @@
+{
+  "$schema": "../../ibc_data.schema.json",
+  "chain_1": {
+    "chain_name": "terra2testnet",
```

Comment:
terra2testnet must be chain_2

Reply:
just swapped them, thanks for catching that!